cap-05/src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Videos</title>
+		<meta charset="utf-8"/>
+	</head>
+    <body>
+    
+        <h1>Vídeo y audio</h1>
+        
+        <video width="640" height="480" controls>
+            <source src="videos/elsenor.mp4" type='video/mp4; codecs="avc1.42E01E, mp4a.40.2"'>
```

Comment:
y hay que separar los atributos cuando sean más de uno

https://github.com/semanticmx/html5-coding-standards?tab=readme-ov-file#proper-use-of-standards-in-html5